graylog2-web-interface/src/components/common/ContentPackMarker.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const ContentPackMarker = React.createClass({
+  propTypes: {
+    contentPack: React.PropTypes.string.isRequired,
+    marginLeft: React.PropTypes.number,
+    marginRight: React.PropTypes.number,
+  },
+
+  getDefaultProps() {
+    return { marginLeft: 0, marginRight: 0 };
+  },
+
+  render() {
+    const style = { marginLeft: this.props.marginLeft, marginRight: this.props.marginRight };
```

Comment:
@edmundoa What do you think, would it be worthwhile to give ContentPackMarker either two components as props or two children to get rid of the inline styles?
Or is that overkill?

Also see comment below for an example.